build.sh

```diff
@@ -1,3 +1,8 @@
-docker-compose -f docker-compose.local.yml rm;
+#!/bin/bash
```

Comment:
@karlfloersch Can you check the logic here to see if it makes sense? The `build.sh` and `test.sh` scripts